src/components/Rockets.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Rocket from './Rocket';
+import { getRockets } from '../redux/actions/rockets';
+
+function Rockets() {
+  const rockets = useSelector((state) => state.rockets);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!rockets.length) {
+      dispatch(getRockets());
+    }
+  }, []);
+
+  return (
+    <>
+      {
+        rockets.map((rocket) => (
+          <Rocket
+            key={rocket.id}
+            id={rocket.id}
+            image={rocket.image[0]}
+            name={rocket.name}
+            description={rocket.description}
+            reserved={rocket.reserved}
```

Comment:
[OPTIONAL] Read my comment on destructuring. [You can read more about the Best practices of JS here. ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) :slightly_smiling_face: